bcda/cclf/metrics/metrics.go

```diff
@@ -146,7 +147,7 @@ type noopTimer struct {
 }
 
 func (t *noopTimer) new(parentCtx context.Context, name string) (ctx context.Context, close func()) {
-	return context.Background(), noop
```

Comment:
I changed this to use the parentCtx because it creates a new nil context when there's not a newrelic timer; this overwrites the logger passed around in the context. this only occurs when the license key is not set and should affect local development only.